tools/tck-integrity-tests/src/main/scala/org/opencypher/tools/tck/FeatureFormatChecker.scala

```diff
@@ -160,10 +163,6 @@ class FeatureFormatChecker extends TCKCucumberTemplate {
     }
   }
 
-  private val scenariosWithIntentionalStyleViolations =
-    Set("Keeping used expression 3",
-        "Keeping used expression 4")
-
```

Comment:
Nice improvement moving these out to the scenarios themselves. Also improves understandability of scenarios